diff -urN qtbase.orig/src/plugins/platforms/windows/qwindowsmousehandler.cpp qtbase/src/plugins/platforms/windows/qwindowsmousehandler.cpp
--- old/qtbase.orig/src/plugins/platforms/windows/qwindowsmousehandler.cpp	2019-04-09 06:52:04.000000000 +0200
+++ new/qtbase/src/plugins/platforms/windows/qwindowsmousehandler.cpp	2022-04-06 22:02:54.748120001 +0200
@@ -60,32 +60,32 @@
  * are present in the Windows SDK's, but not in older MSVC Express
  * versions. */
 
-#if defined(Q_CC_MINGW) || !defined(TOUCHEVENTF_MOVE)
+//#if defined(Q_CC_MINGW) || !defined(TOUCHEVENTF_MOVE)
+//
+//typedef struct tagTOUCHINPUT {
+//    LONG x;
+//    LONG y;
+//    HANDLE hSource;
+//    DWORD dwID;
+//    DWORD dwFlags;
+//    DWORD dwMask;
+//    DWORD dwTime;
+//    ULONG_PTR dwExtraInfo;
+//    DWORD cxContact;
+//    DWORD cyContact;
+//} TOUCHINPUT, *PTOUCHINPUT;
+//typedef TOUCHINPUT const * PCTOUCHINPUT;
 
-typedef struct tagTOUCHINPUT {
-    LONG x;
-    LONG y;
-    HANDLE hSource;
-    DWORD dwID;
-    DWORD dwFlags;
-    DWORD dwMask;
-    DWORD dwTime;
-    ULONG_PTR dwExtraInfo;
-    DWORD cxContact;
-    DWORD cyContact;
-} TOUCHINPUT, *PTOUCHINPUT;
-typedef TOUCHINPUT const * PCTOUCHINPUT;
-
-#  define TOUCHEVENTF_MOVE 0x0001
-#  define TOUCHEVENTF_DOWN 0x0002
-#  define TOUCHEVENTF_UP 0x0004
-#  define TOUCHEVENTF_INRANGE 0x0008
-#  define TOUCHEVENTF_PRIMARY 0x0010
-#  define TOUCHEVENTF_NOCOALESCE 0x0020
-#  define TOUCHEVENTF_PALM 0x0080
-#  define TOUCHINPUTMASKF_CONTACTAREA 0x0004
-#  define TOUCHINPUTMASKF_EXTRAINFO 0x0002
-#endif // if defined(Q_CC_MINGW) || !defined(TOUCHEVENTF_MOVE)
+//#  define TOUCHEVENTF_MOVE 0x0001
+//#  define TOUCHEVENTF_DOWN 0x0002
+//#  define TOUCHEVENTF_UP 0x0004
+//#  define TOUCHEVENTF_INRANGE 0x0008
+//#  define TOUCHEVENTF_PRIMARY 0x0010
+//#  define TOUCHEVENTF_NOCOALESCE 0x0020
+//#  define TOUCHEVENTF_PALM 0x0080
+//#  define TOUCHINPUTMASKF_CONTACTAREA 0x0004
+//#  define TOUCHINPUTMASKF_EXTRAINFO 0x0002
+//#endif // if defined(Q_CC_MINGW) || !defined(TOUCHEVENTF_MOVE)
 
 QT_BEGIN_NAMESPACE
 